orchestration/values.yaml

```diff
@@ -20,4 +20,4 @@ argoTemplates:
   softDeleteTable:
     create: true
     name: soft-delete-table
-parallelism: 4
+parallelism: 8
```

Comment:
❓ I am curious; Looking at https://cloud.google.com/compute/docs/machine-types I see that an `n1-standard-4` has 4 hyper-threads, why the bump to parallelism 8 here given that capacity constraint?

Reply:
This parallelism is for Argo, it refers to the number of pods a single workflow is allowed to spin up at a time.

Reply:
An example of it being used downstream: https://github.com/DataBiosphere/hca-ingest/blob/7b565ae53c37feb802a9166109a3c8c8fc4fce33/orchestration/templates/import-hca-total.yaml#L26